test/rubygems/test_gem_commands_install_command.rb

```diff
@@ -130,8 +130,8 @@ def test_execute_local_transitive_prerelease
   end
 
   def test_execute_no_user_install
-    skip 'skipped on MS Windows (chmod has no effect)' if win_platform?
-    skip 'skipped in root privilege' if Process.uid.zero?
+    skip 'skipped on mwsin (chmod has no effect)' if vc_windows?
+    skip 'skipped in root privilege' if Process.uid.zero? && !win_platform?
```

Comment:
> What is the significance of VC Windows?

I can't easily locally test on it, but I guess I could check it with trunk on Appveyor.  The existing tests skipped for Windows, I allowed them to run with MinGW.

> so your logic actually does the opposite of intended

Since there was already a skip for Windows, I assumed that skip was for other operating systems.  Hence, the `&& win_platform?`.  The tests did pass on Travis...

Reply:
Yes, Travis is running on Linux so both conditions are true, so it works as intended. On Windows the `!win_platform` is not true, so the conditional is false.

```
# Powershell
irb(main):001:0> Process.uid.zero? && !Gem.win_platform?
=> false
```

Reply:
I guess my main point, after fumbling around, is that this bit should probably just be left alone because whether it's VC or mingw doesn't matter for chmod.

I wonder if you're thinking of Cygwin (?).

Reply:
From the other PR, I was in a test file and looking at windows skips.

Thinking that some might be rather old, I looked at the whole test suite.  Commented out the skips, and some passed, some didn't.  This PR was the result of that.  Some (but not all) of the tests involving `chmod` did pass.

> Yes, Travis is running on Linux

Currently, standard 'Travis' is Ubuntu and/or MacOS.  I mostly use MacOS for OpenSSL related issues, because Travis Ubuntu uses OpenSSL 1.0.1.  Ruby MinGW builds (2.5 & trunk) are using OpenSSL 1.1.1.  2.2 thru 2.4 use 1.0.2.

> I wonder if you're thinking of Cygwin (?)

At present, I only use MinGW builds locally.  Before my desktop MB self-destructed, I did build mswin locally...